app/views/videos/index.html.erb

```diff
@@ -0,0 +1,3 @@
+<% @video_articles.each do |video_article| %>
+  <%= render "articles/video_player", meta_tags: false, article: video_article %>
```

Comment:
This is a good thought, but we've switched on the main feed to just showing the image which click through and land you on the page, rather than the literal player, which is a pretty heavy burden on the page.

This is the index code, note the video url as the BG image and the video-camera.svg icon and duration. The icon is probably not necessary on a page made only of images.

```
  <% if story.video.present? && story.video_thumbnail_url.present? %>
    <a class="single-article-video-preview" style="background-image:url(<%= story.cloudinary_video_url %>)" href="<%= story.path %>">
      <div class="single-article-video-duration">
        <img src="<%= asset_path("video-camera.svg") %>" /><%= story.video_duration_in_minutes %></div>
    </a>
  <% end %>
```

So basically this will be a grid full of videos and their titles under them. We can fiddle with design.

Reply:
oh! that makes much more sense than having them be play-able. I don't know why I was trying to do that, esp cause the controls weren't showing up. Glad I pushed this sooner than later.

Reply:
![](https://cl.ly/720058d0d3fc/download/Image%202019-04-03%20at%204.51.45%20PM.png)

![](https://cl.ly/0f45b3e6d118/download/Image%202019-04-03%20at%204.54.33%20PM.png)

![](https://cl.ly/f4878370bea6/download/Image%202019-04-03%20at%204.52.34%20PM.png)